### PR TITLE
Update to eslint 8

### PR DESCRIPTION
**Pull request by jpsc**
- Updated to eslint 8
- Updated babel parser
- Remove jQuery and mocha globals

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-        'plugin:vue/recommended',
-        'prettier/vue',
-        'plugin:prettier/recommended'
+        '@nuxtjs',
+        'plugin:prettier/recommended',
+        'plugin:jest-playwright/recommended'
     ],
     rules: {
         quotes: ['warn', 'single', { avoidEscape: true }],
-        'no-console': ['warn'],
```
Comment (jpsc) [inline index.js:10]: This is now part of @nuxt config and it's based on process.env. Warn for development, error for production.

File: index.js [hunk 2]
```diff
@@ -17,26 +19,8 @@ module.exports = {
             }
         ]
     },
-    parser: 'vue-eslint-parser',
-    parserOptions: {
-        parser: 'babel-eslint',
-        ecmaVersion: 2020
-    },
-    env: {
```
Comment (jpsc) [inline index.js:25]: Most of these are part of the extended config's except for jquery and mocha.
If you want those you can keep using the current version of the config.

**Review by bartderijk — DISMISSED** [review]

**Comment by jpsc** [comment]
@bartderijk this is okay to merge and release?

**Comment by bartderijk** [comment]
> @bartderijk this is okay to merge and release?
There appear to be some merge conflicts, but apart from that I'd say it's good to go.

**Comment by jpsc** [comment]
> > @bartderijk this is okay to merge and release?
> > There appear to be some merge conflicts, but apart from that I'd say it's good to go.

Yes, it's fixed. You were too fast, I was taking care of it. You need to approve again 😇

**Review by bartderijk — APPROVED** [review]

**Pull request merged**